### ISSUE TITLE
Add jump links to all pages

### ISSUE THREAD

**Issue by jvine** [projectblacklight/arclight#840]
Part of #820 

Example implementation here: https://searchworks.stanford.edu - I copied this HTML and CSS from there and adjusted it a bit for visual style; but if there's a better way to implement this, feel free.

- [ ] **Add a skip link section as the first div in the `<body>` tag.**

   ```
   <div id="skip-link">
       <a href="#search_field" class="element-invisible element-focusable" data-turbolinks="false">Skip to search</a>
    <a href="#main-container" class="element-invisible element-focusable" data-turbolinks="false">Skip to main content</a>
   </div>
   ```
"Skip to search" and "Skip to main content" should be present on all pages. 

- [ ] **Style the links so they are invisible until they receive focus via tab key.**

   ```
   #skip-link .element-invisible {
       position: absolute !important;
       clip: rect(1px, 1px, 1px, 1px);
       overflow: hidden;
       height: 1px;
   }
   #skip-link {
       position: absolute;
       top: 5px;
       left: 10px;
       z-index: 1;
   }
   #skip-link .element-invisible.element-focusable:active, #skip-link .element-invisible.element-focusable:focus {
       position: static !important;
       clip: auto;
       overflow: visible;
       height: auto;
   }
   #skip-link a:focus {
       background-color: white;
       border-radius: 0 0 0.2rem 0.2rem;
       padding: .5rem 1rem;
   }
   ```

- [ ] **Add a third link on search results pages**

   `<a href="#documents" class="element-invisible element-focusable" data-turbolinks="false">Skip to first result</a>`

**Comment by mejackreed** [comment]
I wonder if this is just a Blacklight thing we can also do.

**Comment by estelendur** [comment]
@mejackreed you mean add it to Blacklight, or already exists in Blacklight? The Blacklight demo site does not appear to have skip links.

**Comment by mejackreed** [comment]
@estelendur sorry for not being more clear. I mean add it to Blacklight.

**Comment by estelendur** [comment]
See [Blacklight issue](https://github.com/projectblacklight/blacklight/issues/2144)
